torch/csrc/jit/runtime/register_cuda_ops.cpp

```diff
@@ -36,14 +36,14 @@ RegisterOperators const reg({
         },
         aliasAnalysisFromSchema()),
     Operator(
-        "cuda::_current_device() -> int",
+        "aten::current_device() -> int",
```

Comment:
we don't want an aten operator called `current_device` -- there is no API (at least in PyTorch eager) for a "current device".

Reply:
Agreed, I have fixed this now.